docs-kits/kits/PCF Exchange Kit/Software Development View/page_software-development-view.md

```diff
@@ -151,18 +157,28 @@ The following JSON shows the the EDC Asset for PCF defined in the EDC using the
 }
 ```
 
+</details>
+
 #### Payload for EDC Policy
 
-The following JSON is an policy definition including the policiy credentials for "active membership", "frameworkagreement pcf" and "pcf base usage purpose". The [frameworkagreement document](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_PCF.pdf) is published via the association and are available via the Catena-X homepage. The membership credential is automatically created after finishing successfully the onboarding process.
+The following JSON is a policy definition including the *mandatory* policy constraints for "active membership", "signed frameworkagreement" and "pcf base usage purpose". The [framework agreement document](https://catena-x.net/fileadmin/user_upload/04_Einfuehren_und_umsetzen/Governance_Framework/231016_Catena-X_Use_Case_Framework_PCF.pdf) is published via the association and available via the Catena-X homepage. The membership credential is automatically created after successfully finishing the onboarding process.
+
+In addition an *optional* constraint for an existing tenant-specific bilateral contract can be added. It should reference the contract number and is NOT backed up by verifiable credentials. Nevertheless it will be checked on a syntactical level during contract negotiation. If such a contract refererence should be used, a specific usage policy per tenant is needed, as constraint concatenation via OR is not recommended.
+
+>**Note**
+>More informations can be found in the official [ODRL Profile Definition](https://github.com/catenax-eV/cx-odrl-profile/blob/main/profile.md).
```

Comment:
(Only if a specific version of the references file is required, ) please use a static link for files that are maintained within github to prevent a dynamic change of content: https://github.com/catenax-eV/cx-odrl-profile/blob/c96b426ded7922f44bdc33bfc5e0bf754031b5e8/profile.md

Reply:
It meant to be a general note that this is a source of information. No specific version required.